type/__download/gencode-local

```diff
@@ -39,7 +39,7 @@ else
     exit 1
 fi
 
-echo "download_tmp=\"\$( mktemp )\""
+echo "download_tmp=\"$__object/files/download_file\""
```

Comment:
To follow same style, maybe `.../files/download_tmp` instead?

Reply:
I thought about the exact same name, but also thought that it will rest there a long time so it won't be "that temporary".

(Cause my thoughts with disk space - we can remove it after usage. Then it's only temporary. Will save space in the cache, but hurt theoretical consistency.)

Reply:
Currently we delete temp file - see last line of `gencode-local`. Also when sum fails.

And we need to `echo "mkdir '$__object/files'"` too, just before `echo "download_tmp=...`.

Reply:
@Matze1224 did we get too hardcore? :smile: 

Let me know how can I help you with this PR.

Reply:
no, just needed to get time and attention for it :-)